### PR TITLE
UIP-882: add labelWidth prop

### PR DESCRIPTION
**Pull request by rey-wright**
## Description
Add labelWidth prop (fairly straightforward)

## Screenshots
Screen Shot 2020-07-22 at 9 31 51 AM

## Checklist
- [x] Docs have been rebuilt (`yarn build:full`)
- [x] All unit tests pass
- [x] Snapshots have been updated
- [x] No lint errors or warnings have been introduced
- [x] **[Version bumped](https://github.com/cbinsights/form-design-system#updating-version-number) if appropriate**

### CODE REVIEW

File: src/components/TextInput/index.jsx [hunk 1]
```diff
@@ -77,6 +78,7 @@ const TextInput = React.forwardRef(
           {showLabel && label && (
             <div
               className={cc([{ 'fontWeight--bold': isLabelBold }, 'fdsTextInput-label'])}
+              style={labelWidth ? { width: labelWidth } : undefined}
```
Comment (jjniczCB) [inline src/components/TextInput/index.jsx:81]: [could] --add a unit test

**Comment by jjniczCB** [comment]
FYI @akdetrick @rey-wright, DP has been using [could], [should], [must] in reviews lately to specify the urgency of a comment

File: src/components/TextInput/index.jsx (hunk 1 above)
Comment (jjniczCB) [reply src/components/TextInput/index.jsx:81]: nvm didn't see snapshot

**Comment by rey-wright** [comment]
@jjniczCB interesting. any articles / resources behind that idea? Or purely a creation of your own?

**Comment by jjniczCB** [comment]
> @jjniczCB interesting. any articles / resources behind that idea? Or purely a creation of your own?

@kchen-cbi did you have a source for this?

**Comment by kchen-cbi** [comment]
@rey-wright @jjniczCB 
It was inspired by this post
https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify/
and supposedly already in use by our Data team.

**Comment by rey-wright** [comment]
@kchen-cbi seems very interesting, I really like their system.

**Pull request merged**